kivy/core/window/_window_sdl2.pyx

```diff
@@ -89,28 +95,39 @@ cdef class _WindowSDL2Storage:
 
         SDL_SetHint(SDL_HINT_ACCELEROMETER_AS_JOYSTICK, b'0')
 
+        SDL_SetHintWithPriority(b'SDL_ANDROID_TRAP_BACK_BUTTON', b'1',
```

Comment:
As far as I can tell from the code, SDL2 doesn't care if we set this even in older versions where the hint doesn't exist. I didn't explicitly test this though.